packages/now-cgi/index.js

```diff
@@ -3,13 +3,16 @@ const { mkdirp, copyFile } = require('fs-extra');
 
 const glob = require('@now/build-utils/fs/glob');
 const download = require('@now/build-utils/fs/download');
-const { createLambda } = require('@now/build-utils/lambda');
+const {
+  createLambda,
+  getLambdaOptionsFromFunction,
+} = require('@now/build-utils/lambda');
```

Comment:
Let's change this to read from the root export

```js
const {
  glob,
  download,
  createLambda,
  getLambdaOptionsFromFunction,
} = require('@now/build-utils');
```

Reply:
👍